bnd-maven-plugin/src/main/java/aQute/bnd/maven/plugin/BndMavenPlugin.java

```diff
@@ -57,13 +58,16 @@
 
 	@Parameter(defaultValue = "${project.build.outputDirectory}", readonly = true)
 	private File classesDir;
-	
+
 	@Parameter(defaultValue = "${project.build.outputDirectory}/META-INF/MANIFEST.MF", readonly = true)
 	private File manifestPath;
-	
+
 	@Parameter(defaultValue = "${project}", required = true, readonly = true)
 	private MavenProject project;
-	
+
+	@Component
+	private BuildContext buildContext;
```

Comment:
OK I reread that wiki page and the MUST only applies to cases where plugins want to change workspace resources (we don't) or they want to skip some incremental builds based on uninteresting changes (we don't), so I'll remove that.